sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -198,7 +199,8 @@ async function findWorkspaceRoot(programDirectory: string): Promise<string | nul
         }
         const workspaces = parseWorkspaces(p);
         for (const workspace of workspaces) {
-            const files = await pGlob(upath.join(currentDir, workspace, "package.json"));
+            const globber = new fdir().withBasePath().glob(upath.join(currentDir, workspace, "package.json"));
```

Comment:
Odd that both glob and crawl take the currentDir?

Reply:
It looks a little odd, but that's how the library works. The call to `glob` configures a filter. When `crawl` is called, it will check each result to see if it matches the glob.